### ISSUE TITLE
How do you train your scene model?

### ISSUE THREAD

**Issue by he010103** [hujie-frank/SENet#30]
I found out that you use pretrained model of scene in https://github.com/lishen-shirley/Places2-CNNs.git.I also use this pre-trained model  with the se block. But the accuracy of my model can not exceed that of the origin models without se block. Can you tell me more details about training with places 365 data? Do you add auxiliary loss as mentioned in 《Relay Backpropagation for Effective Learning of Deep Convolutional Neural Networks》？ @hujie-frank

**Comment by hujie-frank** [comment]
The training settings are basically the same as the ImageNet-1k dataset except for more iterations because of more training images.

**Comment by hujie-frank** [comment]
We didn't use additional auxiliary loss and trained ResNet-152 and SE-ResNet-152 from scratch without any pre-trained model. Maybe you can use the [released prototxt](https://github.com/hujie-frank/SENet/tree/master/models) to try again.

**Comment by he010103** [comment]
@hujie-frank  thanks you for your reply. My gpu memory is 8GB. Maybe is too small to train such big model from scratch. So I prefer to train places365 using pre-trained se-resnext50. It may be faster!
